setup/release.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+python ./setup/setup_idaplugin.py sdist --dist-dir ./setup/idaplugin_dist --formats=zip upload -r pypitest
+python ./setup/setup_server.py sdist --dist-dir ./setup/server_dist --formats=zip upload -r pypitest
+
+# cleanup
+rm -r ./setup/idaplugin_dist ./setup/server_dist ./rematch_idaplugin.egg-info ./rematch-server.egg-info
```

Comment:
I guess I'm going back to my original rant, this wouldn't work from the ./setup directory 
unless a simple check will be added or something will verify the path.

Directories should be relative and reachable, so I think a path switch might be relevant here 
or a check that this script isn't run from the ./setup directory.

Also, bash, this wouldn't run on Windows : /